### ISSUE TITLE
Question: all loxone services implemented?

### ISSUE THREAD

**Issue by davidspr** [CONNCTED/homebridge-loxone#4]
Does the homebridge-loxone plugin actually implement all Loxone equipment (lights, shades,...) or only the few supported device types temperature, humidity and air quality sensors, outlets (?) and the newly implemented alarm (security system)?
From my point of view it would be of great benefit to implement and allow to connect all Loxone services in this plugin - is this realistic?

**Comment by geertsmichael** [comment]
Yes, I am planning to support more and more Loxone building blocks.
Supporting all Loxone services will take a lot of time, feel free to help and open a pull request to contribute :)

**Comment by davidspr** [comment]
Willig to do so with a little guidance!
